cmd2/cmd2.py

```diff
@@ -2459,9 +2459,12 @@ def _print_topics(self, header: str, cmds: List[str], verbose: bool) -> None:
                     doc_block = []
                     found_first = False
                     for doc_line in doc.splitlines():
-                        str(doc_line).strip()
-                        if len(doc_line.strip()) > 0:
-                            doc_block.append(doc_line.strip())
+                        stripped_line = doc_line.strip()
+                        if stripped_line.startswith(':'):
+                            # Stop since we've now hit the :param type lines
+                            break
```

Comment:
What if someone put relevant documentation after the :param lines?

Reply:
How about this?

```Python
# Don't include :param type lines
if stripped_line.startswith(':'):
    if found_first:
        break
elif stripped_line:
    doc_block.append(stripped_line)
    found_first = True
elif found_first:
    break
```

Reply:
Yeah, I think that would work fine.  That way it will retain comments either before the :param block or after, but if both are present it will only retain the comments before.  Correct?

Reply:
Yes, if comments appear before, then found_first is True. So it should break the moment it hits a :param line.